examples/with-ant-design/package.json

```
@@ -6,7 +6,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "@ant-design/cssinjs": "^1.16.1",
+    "@ant-design/cssinjs":  ">=1.21.0",
```

There is an extra space

```suggestion
    "@ant-design/cssinjs": ">=1.21.0",
```

---

Thank you so much!
Fixed!